tests/testthat/test-run.R

```diff
@@ -154,3 +154,19 @@ test_that("clean parameters given spec", {
   expect_error(check_parameters(list(a = 1, c = 3, d = 4), info),
                "Extra parameters: 'c', 'd'")
 })
+
+
+test_that("can run packet with dependency", {
+  path <- test_prepare_orderly_example(c("minimal", "depend"))
+  env <- new.env()
+  id1 <- orderly_run("minimal", root = path, envir = env)
+  id2 <- orderly_run("depend", root = path, envir = env)
```

Comment:
I tried running the `"depend"` task first and we get an error here but not the most transparent one. 

It errors with "Error in private$metadata_load(id) : id 'NA' not found in index" - worth adding a test for the error case and handling with a bit more of a useful message?

Reply:
definitely, thanks